packages/wonder-blocks-modal/components/one-pane-dialog/one-pane-dialog.md

```diff
@@ -93,46 +96,165 @@ const styles = StyleSheet.create({
-                    </Body>
-                    <Body>
-                        {"Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."}
-                    </Body>
                 </View>
             }
-            footer={<Button type="button">Button (no-op)</Button>}
-            above={<View style={styles.above} />}
+            footer={<Footer />}
             onClose={() => alert("This would close the modal.")}
-            style={styles.customModal}
         />
     </View>
 </View>;
+```
+
+### Example: Multi-step flows
+
+```js
+const {StyleSheet} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {Body} = require("@khanacademy/wonder-blocks-typography");
+const {Strut} = require("@khanacademy/wonder-blocks-layout");
+const {LabelLarge} = require("@khanacademy/wonder-blocks-typography");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {MediaLayout} = require("@khanacademy/wonder-blocks-layout");
+
+const styles = StyleSheet.create({
+    previewSizer: {
+        height: 512,
+    },
+
+    modalPositioner: {
+        // Checkerboard background
+        backgroundImage: "linear-gradient(45deg, #ccc 25%, transparent 25%), linear-gradient(-45deg, #ccc 25%, transparent 25%), linear-gradient(45deg, transparent 75%, #ccc 75%), linear-gradient(-45deg, transparent 75%, #ccc 75%)",
+        backgroundSize: "20px 20px",
+        backgroundPosition: "0 0, 0 10px, 10px -10px, -10px 0px",
+
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "center",
+
+        position: "absolute",
+        left: 0,
+        right: 0,
+        top: 0,
+        bottom: 0,
+    },
+});
+
+const defaultStyles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        justifyContent: "flex-end"
+    },
+    footer: {
+        alignItems: "center",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        width: "100%"
+    }
+});
+
+const smallStyles = StyleSheet.create({
+    footer: {
+        justifyContent: "flex-end"
+    },
+    label: {
+        display: "none"
+    }
+});
+
+const styleSheets = {
+    all: defaultStyles,
+    small: smallStyles
+};
+
+<View style={styles.previewSizer}>
+    <View style={styles.modalPositioner}>
+        <MediaLayout styleSheets={styleSheets}>
```

Comment:
Love the use of `MediaLayout` in these examples.  I want push the use of `MediaLayout` more heavily in webapp.  It makes composition of styles easier when dealing with media queries.